Makefile

```diff
@@ -60,7 +60,7 @@ test:
 	@make test-server
 
 test-server:
-	@docker exec -t sc-flask pytest server/
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Consider documenting the reason for pytest '-s' option.

The addition of '-s' option to pytest command enables the display of print statements directly to the console. It's a good practice to document such changes for future reference, especially if this affects how test results are interpreted or debugged.

```suggestion
test-server:
	@echo "Running pytest with -s option to enable print statements in console output for better debugging."
	@docker exec -t sc-flask pytest -s server/
```